homework/index.js

```diff
@@ -30,18 +30,96 @@
     return elem;
   }
 
+  function drawRepository(data, value, container) {
+    // Clear the details of the selected option
+    container.innerHTML = '';
+    drawRepositoryInfo(data, value, container);
+    drawRepositoryContributors(data, value, container);
+  }
+
+  function drawRepositoryInfo(data, value, container) {
+    const leftDiv = createAndAppend('div', container, { class: 'left-div' });
+    const repoTable = createAndAppend('table', leftDiv);
+    const repoTbody = createAndAppend('tbody', repoTable);
+
+    const tr1 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr1, { text: 'Repository:' });
+    const repositoryCell = createAndAppend('td', tr1);
+    createAndAppend('a', repositoryCell, { text: data[value].name, href: data[value].html_url });
+
+    const tr2 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr2, { text: 'Description:' });
+    createAndAppend('td', tr2, { text: data[value].description });
+
+    const tr3 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr3, { text: 'Forks:' });
+    createAndAppend('td', tr3, { text: data[value].forks_count });
+
+    const tr4 = createAndAppend('tr', repoTbody);
+    createAndAppend('td', tr4, { text: 'Update:' });
+    createAndAppend('td', tr4, { text: data[value].updated_at });
+  }
+
+  function drawRepositoryContributors(data, value, container) {
+    const contributorURL = data[value].contributors_url;
+    const rightDiv = createAndAppend('div', container, { class: 'right-div' });
+    createAndAppend('p', rightDiv, { class: 'contributor-header', text: 'Contributions' });
+    const ul = createAndAppend('ul', rightDiv, { class: 'contributor-list' });
+
+    fetchJSON(contributorURL, (err, contributorData) => {
+      contributorData.forEach((entry, index) => {
+        const li = createAndAppend('li', ul, { class: 'contributor-item', tabindex: index });
+        createAndAppend('img', li, { src: entry.avatar_url, height: '48', class: 'contributor-avatar' });
+        const div = createAndAppend('div', li, { class: 'contributor-data' });
+        createAndAppend('div', div, { text: entry.login });
+        createAndAppend('div', div, { text: entry.contributions });
+      })
+    });
+  }
+  // function rightContainer(url, data) {
+  //   const rightDiv = document.getElementsByClassName('right-container');
+  //   rightDiv.innerHTML = '';
+  //   // const paragraph = createAndAppend('p', rightDiv, { text: 'contributions' }, { class: 'contibutors-header' });
+  //   // const unlist = createAndAppend(ul, paragraph, { class: 'contributors-list' });
+  // }
+
   function main(url) {
     fetchJSON(url, (err, data) => {
```

Comment:
We are receiving repositories here, so in order to make it a bit explicet we may call it as what it contains 😃 
```suggestion
    fetchJSON(url, (err, repositries) => {
```